src/slice.cr

```diff
@@ -239,6 +239,68 @@ struct Slice(T)
     pointer(count).copy_to(target, count)
   end
 
+  # Copies the contents of this slice into *target*.
+  #
+  # Truncates if this slice doesn't fit into the destination slice.
```

Comment:
I changed this: It raises when src will not fit into dest, but not when src does not completely fill dest.